R/zchunk_L203.demand_input.R

```diff
@@ -37,9 +37,9 @@ module_aglu_L203.demand_input_DISABLED <- function(command, ...) {
              "L101.Pop_thous_R_Yh",
              "L102.pcgdp_thous90USD_Scen_R_Y"))
   } else if(command == driver.DECLARE_OUTPUTS) {
-    return(c("L203.SectorLogitTables[[ curr_table ]]$data",
+    return(c("L203.SectorLogitTables[[curr_table]]",
```

Comment:
Right, this isn't going to work. You can't return `L203.SectorLogitTables[[curr_table]]$data` (or rather you can, but then that name gets assigned to the data object, and the test code doesn't know what to do with it. Hang on, let me look.